website/docs/docs/build/sample-flag.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "About the sample flag"
+description: "Use the sample flag to lower development time and reduce warehouse spend."
+sidebar_label: "The sample flag"
+pagination_next: null
+pagination_prev: "docs/build/empty-flag"
+---
+
+# About the `--sample` flag
+
+:::note
+
+The `--sample` flag is not currently available for Python models. If the flag is used with a Python model, it will be ignored.
```

Comment:
@QMalcolm confirming this is also the case for snapshots ... and seeds?

Reply:
Seeds can be sampled. I.e. `ref(my_seed)` will sample the seed. A seed itself cannot be created as a sample of its inputs because it doesn't actually have any inputs to sample, it is the root node.

For snapshots, during the creation of a snapshot, its inputs will be sampled if running in sample mode. Also, a snapshot _can be_ sampled as an input to another node.

Reply:
Do we want to specify that out in the body of the page or does the callout need to be used to address it?

Reply:
@matthewshaver I don't think we need to callout anything with regard to snapshots, because they'll just work. @graciegoheen does that sound correct?

As for seeds, I think a callout of something like the following would be nice:
> Seeds will be created as normal, but they will sampled when referenced but down stream nodes